test/test_deconvolutional_layer.h

```diff
@@ -10,29 +10,28 @@
 #include "testhelper.h"
 #include "tiny_dnn/tiny_dnn.h"
 
-using namespace tiny_dnn::activation;
-
 namespace tiny_dnn {
 
 TEST(deconvolutional, setup_tiny) {
   deconvolutional_layer l(2, 2, 3, 1, 2, padding::valid, true, 1, 1,
                           backend_t::internal);
 
   EXPECT_EQ(l.parallelize(), true);          // if layer can be parallelized
-  EXPECT_EQ(l.in_channels(), 3u);            // num of input tensors
+  EXPECT_EQ(l.in_channels(), 1u);            // num of input tensors
```

Comment:
@edgarriba:
`in_channels` were 3 earlier - `{vector_type::data, vector_type::weight, vector_type::bias}`. But now only `{vector_type::data}`.

Essentially I would envision all layers to give in and pass out `data` and `aux` vector types later.